README.md

```diff
@@ -228,7 +231,7 @@ Details about the rule language and how to write rules for the processors can be
 ### Installation
 
 Python should be present on the system, currently supported are the versions 3.6 - 3.9.
```

Comment:
Could you also make the following changes in the `gettings_started.rst` file, please?

Reply:
DONE